src/zenml/artifact_stores/base_artifact_store.py

```diff
@@ -380,30 +458,36 @@ def _register(self) -> None:
         if isinstance(self, LocalFilesystem):
             return
 
+        overloads: Dict[str, Any] = {
+            "SUPPORTED_SCHEMES": self.config.SUPPORTED_SCHEMES,
+        }
+        for abc_method in inspect.getmembers(BaseArtifactStore):
+            if getattr(abc_method[1], "__isabstractmethod__", False):
+                overloads[abc_method[0]] = staticmethod(
+                    _sanitize_paths(self.path)(
+                        getattr(self, abc_method[0]),
+                    )
+                )
```

Comment:
```suggestion
        overloads: Dict[str, Any] = {
            "SUPPORTED_SCHEMES": self.config.SUPPORTED_SCHEMES,
        }
        for abc_method in inspect.getmembers(BaseArtifactStore):
            if getattr(abc_method[1], "__isabstractmethod__", False):
                overloads[abc_method[0]] = staticmethod(
                    _sanitize_paths(self.path)(
                        getattr(self, abc_method[0]),
                    )
                )

        if isinstance(self, LocalFilesystem):
            return
```

Reply:
I'll give it a shot but might hit into issues in out-of-artifact store access with LocalFS (configs, etc.)